src/Flash.tsx

```diff
@@ -4,8 +4,33 @@ import {COMMON, get, SystemCommonProps} from './constants'
 import sx, {SxProp} from './sx'
 import theme from './theme'
 
+const variantAliases = {
+  default: 'info',
+  info: 'info',
+  warning: 'warn',
+  danger: 'error',
+  success: 'success'
+}
+
```

Comment:
This is a slightly different solution than the flashIcon constant in theme-preval, but it keeps the single source of truth on the primitives object.